docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,259 @@ but the format and content is still evolving.
 We hope you will contribute your experiences
 and questions that you have.
 
-No two migrations are alike but these are some general guidelines
-for how to approach the project:
+The Keptn Lifecycle Toolkit uses a different paradigm
+than Keptn v1 and so migration is not a straight-forward process.
```

Comment:
this sentence sounds scary 😿 can we replace with something more positive? Something like "The Keptn Lifecycle Toolkit employs a distinct paradigm from Keptn v1, requiring technical adjustments during the migration process."

Reply:
done